applications/StructuralMechanicsApplication/tests/cpp_tests/test_constitutive_law_utilities.cpp

```diff
@@ -187,9 +195,40 @@ KRATOS_TEST_CASE_IN_SUITE(ConstitutiveLawUtilitiesPolarDecomposition, KratosStru
     U_ref(1, 0) = 0.493239;
     U_ref(1, 1) = 0.798078;
     U_ref(2, 2) = 1.0;
-    
+
     KRATOS_CHECK_LESS_EQUAL(norm_frobenius(R-R_ref), tolerance);
     KRATOS_CHECK_LESS_EQUAL(norm_frobenius(U-U_ref), tolerance);
 }
+/**
+* Check the correct calculation of the CL utilities Hencky and Biot strains
+*/
+KRATOS_TEST_CASE_IN_SUITE(CalculateCharacteristicLength, KratosStructuralMechanicsFastSuite2)
+{
+    // Model part
+    Model current_model;
+    ModelPart& this_model_part = current_model.CreateModelPart("Main");
+
+    // First we create the nodes
+    NodeType::Pointer p_node_1 = this_model_part.CreateNewNode(1, 0.0 , 0.0 , 0.0);
+    NodeType::Pointer p_node_2 = this_model_part.CreateNewNode(2, 1.0 , 0.0 , 0.0);
+    NodeType::Pointer p_node_3 = this_model_part.CreateNewNode(3, 1.0 , 1.0 , 0.0);
+
+    // Now we create the "conditions"
+    std::vector<NodeType::Pointer> element_nodes (3);
+    element_nodes[0] = p_node_1;
+    element_nodes[1] = p_node_2;
+    element_nodes[2] = p_node_3;
+    Triangle2D3 <NodeType> triangle( PointerVector<NodeType>{element_nodes} );
+
+    // Compute CalculateCharacteristicLength
+    const double length = ConstitutiveLawUtilities<3>::CalculateCharacteristicLength(triangle);
+
+    KRATOS_CHECK_NEAR(length,  0.745356, tolerance);
+
+    // Compute CalculateCharacteristicLengthOnReferenceConfiguration
+    const double length_origin = ConstitutiveLawUtilities<3>::CalculateCharacteristicLengthOnReferenceConfiguration(triangle);
+    
+    KRATOS_CHECK_NEAR(length_origin,  0.745356, tolerance);
```

Comment:
could you set an updated configuration such that this method returns a different value?

Reply:
Ok